recipes/newrelic/infrastructure/ohi/mysql/debian.yml

```diff
@@ -8,6 +8,7 @@ repository: https://github.com/newrelic/nri-mysql
 
 dependencies:
   - infrastructure-agent-installer
+  - super-agent
```

Comment:
I think this schema represents both dependencies that are needed.
  - infrastructure-agent-installer
  -
Instead, I think we should have something that supports one of the following. This is more clear to me it's a one of instead of both dependency is needed
  -  [ infrastructure-agent-installer,   super-agent]
  -  infrastructure-agent-installer/super-agent
  - infrastructure-agent-installer || super-agent

Reply:
Done in 2a1cf407d887688c36a3dc886b955efddb28b242 and dae9f698daf2ac7127fa9f50c8a718befdc2866b.